### PR TITLE
User tests

### PR DESCRIPTION
**Pull request by OlgaEvdokimova**
User tests

### CODE REVIEW

File: src/test/java/by/exadel/internship/formTest/FormTest.java [hunk 1]
```diff
@@ -46,18 +46,15 @@ private MockMultipartFile formData() {
     public void givenFormWithFile_checkTestData()
             throws Exception {
 
-        MockMultipartFile file = new MockMultipartFile("file", "text.txt",
-                MediaType.MULTIPART_FORM_DATA_VALUE, "some file".getBytes());
-
```
Comment (ShirshovAlex) [inline src/test/java/by/exadel/internship/formTest/FormTest.java]: Why have u deleted it, i think its better to comment it for the future

Comment (OlgaEvdokimova) [reply src/test/java/by/exadel/internship/formTest/FormTest.java]: I got wrong point. I've  restored and commented this

**Review by ShirshovAlex — DISMISSED** [review]

File: src/test/resources/user-test-data.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
+INSERT INTO "user_detail" (u_id, u_firstname, u_lastname, u_email, u_login, u_password, u_role)
```
Comment (viktarkolbik) [inline src/test/resources/user-test-data.sql:1]: Where this test data is used?

File: src/test/java/by/exadel/internship/userTest/UserTest.java [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
 import java.net.URI;
 import java.util.List;
+import java.util.UUID;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-
+@Sql(scripts = "/user-test-data.sql")
```
Comment (mortal-gpt) [inline src/test/java/by/exadel/internship/userTest/UserTest.java]: If your test requires some specific test data, you need to clean up DB after this tests.

Comment (OlgaEvdokimova) [reply src/test/java/by/exadel/internship/userTest/UserTest.java]: Done. Added test insert and delete data for each test

**Review by viktarkolbik — DISMISSED** [review]

**Review by viktarkolbik — APPROVED** [review]

**Review by ShirshovAlex — APPROVED** [review]

**Pull request merged**